hoomd/md/BendingRigidityMeshForceCompute.h

```diff
@@ -89,6 +89,33 @@ class PYBIND11_EXPORT BendingRigidityMeshForceCompute : public ForceCompute
 
     //! Actually compute the forces
     virtual void computeForces(uint64_t timestep);
+
+    virtual Scalar calcEnergy(unsigned int idx_a,
+                              unsigned int idx_b,
+                              unsigned int idx_c,
+                              unsigned int idx_d,
+                              unsigned int type_id);
```

Comment:
```suggestion
    Scalar calcEnergy(unsigned int idx_a,
                              unsigned int idx_b,
                              unsigned int idx_c,
                              unsigned int idx_d,
                              unsigned int type_id) override;
```
... and repeated changes in similar examples.

In modern C++, classes that override methods should use the override keyword.